starlette_admin/templates/forms/tinymce.html

```diff
@@ -0,0 +1,10 @@
+<div class="{% if error %}{{ field.error_class }}{% endif %}">
```

Comment:
I think we can merge `config` with `field.input_params()` by overriding `input_params()` and adding the `config` attribute. A new form template is not neccessary

Reply:
I think we should keep this conversation here: #379